tests/summarywriter_test.cpp

```diff
@@ -79,6 +80,48 @@ TEST_CASE_METHOD(SummaryWriterTestsFixture, "Writing text summary to file",
       " Project developed during GSoc 2020 ", *f1);
 }
 
+/**
+ * Test Histogram summary.
+ */
+TEST_CASE_METHOD(SummaryWriterTestsFixture, "Writing histogram summary to file",
+                 "[SummaryWriter]")
+{	
+  std::default_random_engine generator;
+  std::normal_distribution<double> default_distribution(0, 1.0);
+  for (int i = 0; i < 10; ++i)
+  {
+    std::normal_distribution<double> distribution(i * 0.1, 1.0);
+    std::vector<double> values;
+    for (int j = 0; j < 10000; ++j)
+      values.push_back(distribution(generator));
```

Comment:
```suggestion
    for (int j = 0; j < 10000; ++j)
    {
      values.push_back(distribution(generator));
    }
```

Reply:
The mlpack style guide doesn't recommend this, but I have no issue with either style.

Reply:
Yeah let's follow the style guideline. It improves readability.